templates/Deepcell_Postprocessing.ipynb

```diff
@@ -144,6 +144,15 @@
     "### Afterwards, we can generate expression matrices from the labeling + imaging data"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Returns:\n",
+    "* cell_size_normalized_data: computed by dividing the marker counts in segmentation_labels by their corresponding cell size, however, if the cell size is zero we leave the marker count as is\n",
+    "* arcsinh_transformed_data: the result of passing each value of cell_size_normalized_data * 100 through the arcsinh function, we add the multiplication of 100 to linearly scale the data beforehand"
```

Comment:
I think this is a bit confusing as written. See if you can reword it to be more clear. Breaking it up into two sentences would probably help.